corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -175,6 +217,10 @@
                             data-bind="click: function(data, event) { archive_loc(event.currentTarget, name(), uuid()) }, visible: show_archive_action_button">
                             {% trans "Archive" %}
                         </button>
+                        <button class="btn btn-danger"
+                            data-bind="click: function(data, event) { delete_loc(event.currentTarget, name(), uuid()) }, visible: show_archive_action_button">
+                            {% trans "Delete" %}
+                        </button>
```

Comment:
Might make sense to show this also on the `show_inactive` version, that is - move it outside this `{% else %}` block.